workflow-generator/src/main/java/com/wipro/ats/bdre/wgen/DotUtil.java

```diff
@@ -25,15 +25,19 @@
  */
 public class DotUtil {
     private static final Logger LOGGER = Logger.getLogger(DotUtil.class);
+    private static final String SHAPE=  " [shape=rectangle,style=\"rounded\",penwidth=3];";
+    private static final String FORK = "fork_";
 
+    private DotUtil(){
+    }
     /**
      * This method creates schematic diagram of oozie workflow using dot language.
      *
      * @param oozieNodes This is a collection containing all oozie nodes to be used in the workflow.
      * @return Method returns String buffer of schematic diagram of oozie workflow.
      */
-    public static StringBuffer getDot(Collection<OozieNode> oozieNodes) {
-        StringBuffer dot = new StringBuffer("digraph{\n" +
+    public static StringBuilder getDot(Collection<OozieNode> oozieNodes) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) The Cyclomatic Complexity of this method "getDot" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)